### DISCUSSION TITLE
Will you buy ibax from the pre-sale?

### DISCUSSION THREAD

**Discussion by btn69527** [IBAX-io/go-ibax#1510] [Soru Cevap - Türkçe]
Do you think ibax will go up? Do you have an investment plan? And why did you choose ibax?

**Comment by xxleretxx** [comment]
İbax to moon

**Comment by Yusufknk** [comment]
İbax'ı ilerleyen zamanlarda güzel yerlerde görebiliriz, bunun için geç kalmayın.
